toolchains/iOS.framework.cmake

```diff
@@ -37,7 +37,7 @@ else()
     endif()
 endif()
 
-set(FRAMEWORKS CoreGraphics CoreFoundation QuartzCore UIKit OpenGLES Security CFNetwork GLKit)
+set(FRAMEWORKS CoreGraphics CoreFoundation CoreLocation QuartzCore UIKit OpenGLES Security CFNetwork GLKit)
```

Comment:
I don't know if this is actually needed because it's only in the sample app?

Reply:
Yes this changes the dependency of the framework, you could change it here instead https://github.com/tangrams/tangram-es/blob/master/toolchains/iOS.cmake#L8.

Reply:
Ah ok yeah I changed it there as well. So I can remove this one?

Reply:
Yes